sumologic-service/0.15.0/README.md

```diff
@@ -0,0 +1,151 @@
+
+<!-- TODO: Uncomment this after the PR to support switching SLI provider is merged -->
+<!-- 4. Configure Keptn to use Sumo Logic SLI provider
+Use keptn CLI version [0.15.0](https://github.com/keptn/keptn/releases/tag/0.15.0) or later.
+```bash
+keptn configure monitoring sumologic --project <project-name>  --service <service-name>
+``` -->
+
+5. Trigger delivery
+```bash
+keptn trigger delivery --project=<project-name> --service=<service-name> --image=<image> --tag=<tag>
+```
+Example:
+```bash
+keptn trigger delivery --project=podtatohead --service=helloservice --image=docker.io/jetzlstorfer/helloserver --tag=0.1.1
+```
+Observe the results in the [Keptn Bridge](https://keptn.sh/docs/0.15.x/bridge/)
+
+# Not supported in the query
+- `fillmissing`
+- `outlier`
+- `timeshift`
+
+Why? Because the API does not support `fillmissing` and `outlier`. `timeshift` is supported but you can't write it in the query like `<my-query> | timeshift`. We plan to support `timeshift` in the future ([issue](https://github.com/vadasambar/sumologic-service/issues/1)) but support for `fillmissing` and `outlier` depends on Sumo Logic (can't do anything until Sumo Logic supports it). 
+
+# Rules for using `quantize`
+Based on https://help.sumologic.com/Metrics/Metric-Queries-and-Alerts/07Metrics_Operators/quantize#quantize-syntax
+1. Use only 1 `quantize` (using `quantize` multiple times in a query leads to error)
+2. Use `quantize` immediately after the metric query before any other operator
+3. Quantize should be strictly defined as `query | quantize to [TIME INTERVAL] using [ROLLUP]` (this differs from how Sumo Logic quantize works. You need to be explicit here. Dropping [TIME INTERVAL] or `using` or `[ROLLUP]` won't work)  
+
+Why so many rules? Because [Sumo Logic API does not support quantize in the query](https://api.sumologic.com/docs/#operation/runMetricsQueries). We have implemented a wrapper
+which mimics quantize which works well if you adhere to 
+
+## Compatibility Matrix
+
+*Please fill in your versions accordingly*
+
+| Keptn Version    | [sumologic-service Docker Image](https://github.com/keptn-sandbox/sumologic-service/pkgs/container/sumologic-service) |
+|:----------------:|:----------------------------------------:|
+|       0.15.0      | keptn-sandbox/sumologic-service:0.15.0 |
+
+## Installation
+
+```bash
+export ACCESS_ID="<your-sumologic-access-id>" ACCESS_KEY="<your-sumologic-access-key>"
+# cd sumologic-service
+helm upgrade --install my-sumo sumologic/sumologic   --set sumologic.accessId="${ACCESS_ID}"   --set sumologic.accessKey="${ACCESS_KEY}"   --set sumologic.clusterName="keptn-sumo"
+
+```
+<!-- TODO: Uncomment this after the PR to support switching SLI provider is merged -->
```

Comment:
Does this lack of support mean that IF this PR is merged and shown on ArtifactHub, in fact it can't be used UNTIL the `keptn configure monitoring` related PR is merged? In other words, does this PR have a dependency on another first being merged?

Reply:
Not necessary but I will add some documentation around this because it is not clear how to do it without the `keptn configure monitoring` in the README. i.e., you need to apply the lighthouse config manually for the current released versions of Keptn ([ref 1](https://github.com/vadasambar/sumologic-service/blob/81c225a1182cb4ce5150b3611e19ecdecbf8b34f/examples/kup.sh#L234), [ref 2](https://github.com/vadasambar/sumologic-service/blob/2f0c55d7bbada1482e23deffeb853e24530c9234/examples/quickstart/lighthouse_config.yaml)). It can be argued that we shouldn't release the artifacthub PR until we have merged the PR for `keptn configure monitoring` but if the PR in the main keptn repo is merged, it'll be released with the latest Keptn version of the CLI and I am not sure if that CLI would be fully compatible with 0.15.0 i.e., I don't want to use CLI for say 0.19.x for 0.15.x.